cmd/maya-apiserver/cstor-operator/cspc/operations.go

```diff
@@ -187,3 +202,30 @@ func (pc *PoolConfig) getCSPWithNodeName(nodeName string) (*apis.NewTestCStorPoo
 	}
 	return nil, errors.New("No CSP(s) found")
 }
+
+// isBDUsable returns no error if BD can be used.
+// If BD has no BDC -- it is created
+// TODO: Move to algorithm package
```

Comment:
:+1: for TODO.. there are other functions also which need to be moved to packages